Content.Shared/Examine/ExamineSystemShared.cs

```diff
@@ -109,6 +110,14 @@ public virtual bool CanExamine(EntityUid examiner, MapCoordinates target, Ignore
             if (EntityManager.GetComponent<TransformComponent>(examiner).MapID != target.MapId)
                 return false;
 
+            // Floofstation edit - if the examined thing is a player, just return true
+            if (examined != null)
+                if (EntityManager.TryGetComponent<ActorComponent>(examined, out var _)
```

Comment:
```suggestion
                if (TryComp<ActorComponent>(examined, out var _)
```

Reply:
Sure thing! I assume TryComp just checks if the component exists, while TryGetComponent also fetches the actual component?

Reply:
It is the same thing